### DISCUSSION TITLE
Duplicate of #2608: How can I change the path of many playlist items at once?

### DISCUSSION THREAD

**Discussion by tracure1337** [DeaDBeeF-Player/deadbeef#2623] [Q&A]
Duplicate of #2608: How can I change the path of many playlist items at once?

example:

Failed to play track: 
/run/user/1000/gvfs/smb-share:server=local.local,share=video/youtube_music/music_for_coding

I need to change the path to 
/home/<username>/localshare/youtube_music/music_for_coding

An option to fix this would be great. Otherwise how would I change this programatically?

Thanks so much for looking into this. Would go a great way to make the playlists portable.
I spent many hours perfecting them.

**Comment by tracure1337** [comment]
Actually it would be amazing if it just asked for the new root path when the error pops up.

**Comment by Oleksiy-Yakovenko** [comment]
> Otherwise how would I change this programatically

You would need to edit `plt_load_int` function in the playlist.c
There you'll find the line
```
pl_add_meta (it, ":URI", uri);
```

So before this line, you would need to add some code, which would change the path in the uri string from the old one to the new one.

**Reply by tracure1337** [reply]
Thanks.I will try :+1:
